tests/testthat/test-compiler-ghpages.R

```diff
@@ -27,6 +27,18 @@ test_that("animint2pages() returns owner/repo string", {
   expect_is(viz_owner_repo, "character")
 })
 
+test_that("check if animint2pages() successfully uploads screenshot", {
+  #print('checking screenshot')
+  file_exists <- tryCatch({
+    gh("GET /repos/:owner/:repo/contents/:path",
+       owner = "siddhesh195", repo = "animint2pages_test_repo", path = "screenshot.png",ref = "gh-pages")
```

Comment:
I think the screenshot should be called Capture.PNG
also I guess it is OK to use gh GET instead of cloning and checking the file exists on your local file system
can you change owner to animint? I'm surprised the test works using siddhesh195 as owner, can you please explain how that works? (I thought that tests involving writing to github repos needed to do it with animint permissions, inside animint org, but here it seems like you are doing it in one of your personal repos?)

Reply:
@tdhock that is the older version of the file. somewhere in the series of commits I made the change to replace hardcoded user name with the user name set in the environment. also, I was getting errors on GitHub Actions initially because I was not importing the gh library. But now the checks pass with those two changes.

Reply:
in R there is not "importing the gh library"
you should either do gh::some_fun which loads the gh package (preferred for use in a package) 
or library(gh) which loads it if necessary, then attaches it to the search() path (but typically library is not great to use in a package, instead used for convenience in user analysis scripts)